cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -173,7 +172,7 @@ func (p *BicepProvider) Plan(
 				p.azCli,
 				p.env.GetLocation(),
 				p.env.GetSubscriptionId(),
-				p.env.GetEnvName(),
+				fmt.Sprintf("%s-%d", p.env.GetEnvName(), time.Now().Unix()),
```

Comment:
Would be happy if we wanted to increase the resolution of the timestamp, I just reached for `.Unix()` to have a number.  When we order these deployments to find the "latest" we don't use these timestamps, we use the time embedded in the deployment metadata.

Reply:
I'm happy with this.